rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java

```diff
@@ -122,6 +123,8 @@ public Operation generateExample(final Class<? extends Operation> opClass) throw
             return addNamedView();
         } else if (operation instanceof If) {
             return ifOperation();
+        } else if (operation instanceof While) {
```

Comment:
Just a quick note, depending on which PR is merged first this will need updating